doc/manual.rst

```diff
@@ -2985,7 +2985,8 @@ Example:
   of "delete-everything", "restart-computer":
     echo "permission denied"
   of "go-for-a-walk":     echo "please yourself"
-  else:                   echo "unknown command"
+  elif defined(windows):  echo "bsod" # optional `elif` must come after `of` branches
```

Comment:
This example is poorly chosen as `defined` is usually used within a `when` statement.

Reply:
changed to something else